jupyterlab/handlers/announcements.py

```diff
@@ -110,7 +110,7 @@ async def __call__(self) -> Awaitable[Tuple[str, Tuple[str, str]]]:
                 trans = translator.load("jupyterlab")
                 return (
                     trans.__(f"A newer version ({last_version}) of JupyterLab is available."),
-                    (trans.__("Open changelog"), f"{JUPYTERLAB_RELEASE_URL}{last_version}"),
+                    (trans.__("Changelog…"), f"{JUPYTERLAB_RELEASE_URL}{last_version}"),
```

Comment:
It is recommended to use action verb for button label.

If we are to use to use the ellipsis, for homogeneity we should also align the button `Open privacy policy`.

Reply:
`View changes` or `View details`?

Reply:
Thoughts about using `More info…` or `Read more…` instead? The word "Changelog" is commonly known among developers, but not all Jupyter users are developers.

Reply:
> Thoughts about using `More info…` or `Read more…` instead

`Learn more...` is another option with a verb (not that I think we have to stick to this rule here)

Reply:
I do not like that "info" is used in "More info".

Reply:
Another point: we should not be prioritising for brevity in English because in localised versions with language packs the equivalents may be much longer.